superset/connectors/sqla/models.py

```diff
@@ -255,7 +255,7 @@ class TimePeriod(enum.Enum):
     day = 60 * 24
 
 
-class Alert(Model):
+class Alert(Model, AuditMixinNullable):
```

Comment:
@qv-pat Does the AuditMixinNullable add in the created by, changed on, created on fields?